bootstrap.py

```diff
@@ -639,6 +639,20 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = ['satellite', 'satellite-capsule']
```

Comment:
Would it make sense to in the PR to also add the older spacewalk packages (like spacewalk-backend) to prevent users from running bootstrap.py against a Spacewalk/Satellite Server/Proxy. There are reports of users (see https://bugzilla.redhat.com/show_bug.cgi?id=1448722) doing this and breaking their Satellite 5 Server/Proxy (as we remove a ton of rhn* packages at the end of the script).